drivers/adc/Kconfig.mcux

```diff
@@ -37,11 +37,13 @@ config ADC_MCUX_LPADC
 	help
 	  Enable the MCUX LPADC driver.
 
+if ADC_MCUX_ADC12 || ADC_MCUX_ADC16 || ADC_MCUX_12B1MSPS_SAR || ADC_MCUX_LPADC
```

Comment:
For the time been, only ADC_MCUX_12B1MSPS_SAR (for RT10xx devices) and ADC_MCUX_LPADC (for RT11xx devices) use with ADC_MCUX_ETC.

Reply:
yes, I know.  Anyway, ADC_MUCX_ETC also depends on HAS_MCUX_ADC_ETC